Fix nixies not updating when seen through Remote View; add German localization

n00mkrad opened the pull request:
Remote View fix: Added `player.render_mode == defines.render_mode.chart_zoomed_in` condition to include remote-viewed surfaces in `eyes_on_surface` assignment loop.

Also added German localization (should've made a separate PR but I hope you don't mind) as there was none yet and I'm a native speaker.

pindab0ter committed:
Merge branch 'develop'

n00mkrad committed:
Rename UPSFriendlyNixieTubeDisplay to UPSFriendlyNixieTubeDisplayFixed

n00mkrad committed:
Update info.json with bumped version number, new title/name/desc

n00mkrad committed:
Update changelog.txt

n00mkrad committed:
Fix nixies not updating when seen through remote view

n00mkrad committed:
Add German translation while I'm at it

n00mkrad committed:
Localization: Use Title Case for item names

pindab0ter reviewed (changes requested):
Hey, thank you for the PR. It looks good and I would like to merge it.

Could you first undo all the name changes that are not required for the fix and move your name from the author field in the `info.json` file to the Credits portion of the `README.md` file?

n00mkrad commented:
> Hey, thank you for the PR. It looks good and I would like to merge it.
> 
> Could you first undo all the name changes that are not required for the fix and move your name from the author field in the `info.json` file to the Credits portion of the `README.md` file?

Sure thing, on it.
I presume the version number bump is fine so I'll keep that.

n00mkrad committed:
Revert info.json changes (made for a fork) for merging back

n00mkrad committed:
Updated readme with credits and more unified formatting

Formatting changes:
- All bullet points uniformly end without a period
- "Usage" section is now numbered, matching the mod portal des., as they are consecutive steps
- Clarified that there are two technologies that can be researched
- Unified single-quote symbols

pindab0ter commented:
Some unsolicited advice, I hope you don't mind: What you could've done is put the changes relating to the fix on a different branch before the naming changes. That way you can request _that_ branch to be merged and have the naming changes not appear in that PR, while still allowing you to maintain your own version.

n00mkrad commented:
> Some unsolicited advice, I hope you don't mind: What you could've done is put the changes relating to the fix on a different branch before the naming changes. That way you can request _that_ branch to be merged and have the naming changes not appear in that PR.

Thanks - I also came to that realization but was already done by that time. Making my own fork and a PR at the same time with the same branch was not that smart.

Give me another minute before merging, just noticed something tiny in readme that needs fixing.

n00mkrad committed:
Update README: Revert Mod Portal link to original mod, fix a line break in credits

n00mkrad commented:
Okay, we should be good to go now.

Some of the formatting was personal preference to a degree (straight `'` instead of curly `’` quotes, previously both were used), if you disagree with that you can change it back - as long as everything's consistent :p

pindab0ter reviewed (changes requested):
Could you revert the `__UPSFriendlyNixieTubeDisplay__` to `__UPSFriendlyNixieTubeDisplayFixed__` changes as well?

n00mkrad commented:
Oh damn of course I missed something... I'll rund a find&replace real quick.

n00mkrad committed:
Revert name change ("Fixed" suffix)

pindab0ter commented:
The changes should be live immediately. Thank you for your contribution!
